src/components/Dialogs/Dialogs.jsx

```diff
@@ -14,24 +15,52 @@ const DialogItem = props => {
   );
 };
 
-const Message = props => {
+const Message = (props) => {
   return <div className={s.dialogs}>{props.message}</div>;
 };
 
-const dialogsElements = (usersData) => usersData.map(dialog => (
-  <DialogItem name={dialog.name} id={dialog.id} />
-));
+const dialogsElements = (usersData) =>
+  usersData.map((dialog) => <DialogItem name={dialog.name} id={dialog.id} />);
 
-const messagesElements = (messagesData) => messagesData.map(message => (
-  <Message message={message.message} />
-));
+const messagesElements = (messagesData) =>
+  messagesData.map((message) => <Message message={message.message} />);
 
 const Dialogs = (props) => {
+  let newMessageBody = props.newMessageBody;
```

Comment:
I think it's time for us to use "destructuring assignment".
Get familiar with the page https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
and "Object destructuring" section in particular.
Then use your new knowledge to transform `let newMessageBody = props.newMessageBody;`
Remember - if the "newMessageBody" variable doesn't get new assignment in the current scope, you should use "const"

Reply:
done